grammar.ebnf

```diff
@@ -20,8 +20,19 @@ StructDecl = "struct", Identifier, "{", { StructField }, "}";
 StructField = Identifier, ":", Type, ";";
 
 
+Statement = "{", { Statement }, "}"
+          | Type, Identifier, ";"
+          | "while", "(", Expression, ")", "{", { Statement }, "}"
+          | Lval, "=", Expression, ";"
+          | Lval, "[", Expression, "]", "=", Expression, ";"
+          | "if", "(", Expression, ")", Statement;
```

Comment:
Please make curly braces obligatory.

Reply:
Is it really necessary? One-liner ifs are quite common and putting obligatory braces seems a bit excessive and makes the code a bit harder to read, especially for people new to programming, who are this language's target audience (even if it's a mock project :p )

Reply:
Well, for me it makes the code harder to read to _not_ put obligatory braces (not in all cases, but in most in my experience).
However it's true that most people seem to like it so I agree, we'll keep it your way.